catroid/src/main/res/values-ja/strings.xml

```diff
@@ -1708,7 +1708,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">比較演算子</string>
     <string name="formula_editor_object_general">全般プロパティ</string>
     <string name="formula_editor_object_movement">運動プロパティ</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations